cdap-docs/reference-manual/source/characters.rst

```diff
@@ -10,22 +10,28 @@
 Supported Characters
 ====================
 
-The Cask Data Application Platform (CDAP) has naming conventions for different components of CDAP.
+The Cask Data Application Platform (CDAP) has different naming conventions for different entities in CDAP.
 
-Streams
--------
+Alphanumeric Characters
+-----------------------
+**Metadata tags** are restricted to the alphanumeric character set:
 
-Stream names can have these characters:
+- Alphanumeric characters (``a-z A-Z 0-9``)
```

Comment:
Fixed. I will see about revising the supported characters so that the descriptions cover all CDAP entities.